pkg/operation/botanist/constraints_check.go

```diff
@@ -87,10 +85,8 @@ func (b *Botanist) CheckHibernationPossible(ctx context.Context, constraint gard
 	}
 
 	for _, webhookConfig := range mutatingWebhookConfigs.Items {
-		for i, w := range webhookConfig.Webhooks {
-			uid := fmt.Sprintf("%s/%s/%d", webhookConfig.Name, w.Name, i)
-			accessor := webhook.NewMutatingWebhookAccessor(uid, webhookConfig.Name, &w)
```

Comment:
Why did we need this earlier?

Reply:
Before, we used the webhook accessor to uniformly handle mutating and validating webhooks.
Now that k8s deps are bumped, `New{Mutating,Validating}WebhookAccessor` can only take webhook configs in version `admissionregistration/v1` which we cannot assume to be present in all Shoot clusters. So I just get the config in version `v1beta1` and pass the relevant fields/slices down to the helper func.

Reply:
Yes, understood. I'm just wondering why we didn't do it like this earlier already, and instead created the webhook accessor. Do you know?

Reply:
I think, I introduced it back then, but with no special reason. :)

Reply:
OK, I see, thank you!